app/src/main/res/values/strings.xml

```diff
@@ -24,12 +24,12 @@
     <string name="home_notice_content">Download Magisk ONLY from the official GitHub page. Files from unknown sources can be malicious!</string>
     <string name="home_support_title">Support Us</string>
     <string name="home_item_source">Source</string>
-    <string name="home_support_content">Magisk is, and always will be, free and open-source. You can however show us that you care by sending a small donation.</string>
```

Comment:
Misspelling of "open source", a term that produces errors in translation.

Reply:
Fair enough, it's a dash though.

Reply:
Nono, it is much worse than that.
If it isn't clear what the user should care about, the likelihood of receiving donations dwindle.
If the requirement is to already know, only people already in the know will have a chance to grasp what they already likely know. In my experience this is the main point to get right if you want donations.

Reply:
Feel free to workshop the donations part more, but the first sentence needs to be restored per John's wishes.